playbooks/tasks/lastpass-save-cluster-config.yml

```diff
@@ -49,10 +49,44 @@
           and your config files and then rerun this playbook.
       when: update_lastpass_result.user_input[:1] not in 'yY'
 
+    - name: Keep better yaml formatting for mongo_db_mgkey
+      block:
+        - name: Store mongo mgkey to temporary variable and indent it
+          vars:
+            # Prefix that will be rendered in final LastPass yaml
+            prefix: "  "
+          set_fact:
+            # Add prefix to each mgkey line
+            indented_mgkey: >-
+              {{ [prefix]
+              | product(webportal_common_config_last.mongo_db_mgkey.splitlines())
+              | map('join')
+              | list
+              | join('\n') }}
+
+        - name: Update last config with a dummy mongo mgkey value
```

Comment:
REQ:  This needs a comment. Why are we adding a dummy value to the mgkey value?

Reply:
Added an explanation comment before the whole block incl. examples.